driver-mapping/src/test/java/com/datastax/driver/mapping/MapperAccessorParamsTest.java

```diff
@@ -85,6 +131,43 @@ void should_include_enumtype_in_accessor_string() {
         ResultSet addUserStr(int key, @Enumerated(EnumType.STRING) Enum value);
     }
 
+    /** Tests various ways to match method parameters to query bind markers. */
+    @Accessor
+    public interface UserPhoneAccessor {
+        /** Standard positional markers */
+        @Query("update user set home_phone = ?, work_phone = ? where key = ?")
+        void updatePhones_positional(String homePhone, String workPhone, int key);
+
+        /** Standard named markers */
+        @Query("update user set home_phone = :home, work_phone = :work where key = :key")
```

Comment:
Yes, will do that.
Sorry I always forget to run the tests against older C* versions.